README.md

```diff
@@ -1,6 +1,10 @@
 # openc-schema
 
-JSON Schema to validate data before sending to OpenCorporates.
+JSON Schema to validate data before sending it to be **ingested into** OpenCorporates.
+
+Originally these files were designed for use by external developers, along with the [openc_bot gem](https://github.com/openc/openc_bot),  with OpenCorporates encouraging anyone to join in with writing bots for data gathering and ingestion. Lately we have moved away from that approach, and all meaningful use of these schema files is confined to bots living in our own private repos.
```

Comment:
Note: I added similar sentence to the openc_bot gem README https://github.com/openc/openc_bot#external-developer-documentation  (also public)